LICENSE

```diff
@@ -387,6 +387,12 @@ This project includes the software: Swagger UI
   Used under the following license: Apache License, version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
   Copyright (c) SmartBear Software (2011-2015)
 
+This project includes the software: Tango Project Icons
+  Available at: https://commons.wikimedia.org/wiki/File:Network-server.svg
+  Inclusive of: server.svg
+  Used under the following license: Public Domain (https://en.wikipedia.org/wiki/en:public_domain)
+  Released into the public domain by the people from the Tango! project (The Tango! Desktop Project), via Wikimedia Commons
+
```

Comment:
I think [Richards PR](https://github.com/apache/brooklyn-docs/pull/167) added this.